engine/src/main/java/org/terasology/world/WorldProvider.java

```diff
@@ -29,19 +31,49 @@
     /**
      * An active block is in a chunk that is available and fully generated.
      *
-     * @param pos
+     * @param pos The position
      * @return Whether the given block is active
+     * @deprecated This is scheduled for removal in an upcoming version
+     *             method will be replaced with JOML implementation {@link #isBlockRelevant(Vector3ic)}.
      */
+    @Deprecated
     boolean isBlockRelevant(Vector3i pos);
 
+    /**
+     *
+     * @param pos The position
+     * @return Whether the given block is active
+     */
+    boolean isBlockRelevant(Vector3ic pos);
```

Comment:
Should we consider (in general) adding default interface implementations when adding new methods like this? That could avoid having to update _some_ things in modulespace. In this case I'm not sure it makes sense as I only see one non-engine usage over in FlexiblePathfinding.

Reply:
It can't hurt at thh moment but a lot of this is temporary and I all the old methods are going to go away anyways.